superdesk/io/commands/update_ingest.py

```diff
@@ -628,11 +628,12 @@ def ingest_item(item, provider, feeding_service, rule_set=None, routing_scheme=N
             _ingest_cancel(item, feeding_service)
 
         rend = item.get("renditions", {})
+        request_kwargs = feeding_service.get_request_kwargs()
```

Comment:
I think it might be useful eventually to pass the service instance to those methods, there could be different ways of downloading renditions which we wouldn't be able to handle via request args only